src/Xablu.Walkthrough/Plugin.Xablu.Walkthrough.Android/Resources/layout/theme_forestprime_page.axml

```diff
@@ -4,43 +4,48 @@
     android:id="@+id/theme_forestprime_page"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
-    <TextView
-        android:id="@+id/theme_forestprime_title"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:gravity="center"
-        android:textStyle="bold"
-        android:layout_marginTop="60dp" />
-    <LinearLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:gravity="center"
-        android:orientation="vertical">
-        <ImageView
-            android:id="@+id/theme_forestprime_image"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:scaleType="centerCrop"
-            android:adjustViewBounds="true"
-            android:paddingLeft="16dp"
-            android:paddingRight="16dp" />
-        <TextView
-            android:id="@+id/theme_forestprime_description"
+        android:layout_height="match_parent">
+        <Button
+            android:id="@+id/theme_forestprime_gettingstarted"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            android:layout_height="70dp"
```

Comment:
this is a hardcoded height? That cannot be adjusted later it seems

Reply:
It's hard to see in this but this is the height of the "Get Started" button at the bottom of the page. It's the same height as it was before.